_targets.R

```diff
@@ -20,17 +20,17 @@ yaml_file = 'test_config.yml'
 
 # Set up python virtual environment ---------------------------------------
 
-if(!dir.exists('env')) {
-  source('data_acquisition/src/pySetup.R')
+if (!dir.exists("env")) {
+  source("data_acquisition/src/pySetup.R")
 } else {
-  use_condaenv(file.path(getwd(), 'env'))
+  use_condaenv(file.path(getwd(), "env"))
 }
 
 
 # Source functions --------------------------------------------------------
 
-source('data_acquisition/src/general_functions.R')
-source_python('data_acquisition/src/gee_functions.py')
+source("data_acquisition/src/general_functions.R")
```

Comment:
Not sure it would even be helpful, but just a note that an alternative here could be `tar_source()`

Reply:
Amazing. Will implement.